### PR TITLE
feat(setup): create artificial tweet component

### PR DESCRIPTION
**Pull request by sunanmau5**
- create artificial tweet component using direflow
- add generate tweet UI component
- add ability to copy generated tweet

image

### CODE REVIEW

**Review by NathanaelHermanto — APPROVED** [review]

File: packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+export const Wrapper: React.FunctionComponent = () => {
+  const [party, setParty] = useState<Party>(Party.parteilos);
+  const [generatedTweet, setGeneratedTweet] = useState<string>('');
+  const [isCopied, setIsCopied] = useState<boolean>(false);
+  const onCopyText = () => {
+    setIsCopied(true);
+    setTimeout(() => setIsCopied(false), 1000);
+  };
+
+  const generateTweet = useCallback(() => {
```
Comment (LouisAndrew) [inline packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx]: Why do we need usecallback here?

File: packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            ))}
+          </Select>
+        </FormControl>
+        <FormControl sx={{ m: 1, minWidth: 120 }}>
+          <TextField label="Keyword" variant="outlined" />
+        </FormControl>
+      </Box>
+      {generatedTweet && (
+        <Box sx={{ display: 'flex' }}>
+          <Typography sx={{ mr: 1 }}>{generatedTweet}</Typography>
```
Comment (LouisAndrew) [inline packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx]: Would be nice to have an indicator that this is the generated tweet

File: packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+              <MenuItem key={key} value={value}>{value}</MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+        <FormControl sx={{ m: 1, minWidth: 120 }}>
+          <TextField label="Keyword" variant="outlined" />
+        </FormControl>
+      </Box>
+      {generatedTweet && (
+        <Box sx={{ display: 'flex' }}>
```
Comment (LouisAndrew) [inline packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx]: Maybe add some padding?

File: packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  TextField,
+  Typography,
+} from '@mui/material';
+import React, { useCallback, useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```
Comment (LouisAndrew) [inline packages/artificial-tweet/src/direflow-components/artificial-tweet/components/wrapper.tsx]: Food for thought: do we really need the `CopyToClipboard` library? https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API? Whatever you choose it's fine

**Review by LouisAndrew — CHANGES_REQUESTED** [review]

**Review by LouisAndrew — APPROVED** [review]
🍕🍕

**Pull request merged**